src/gui/tray/UserModel.cpp

```diff
@@ -907,12 +915,13 @@ QHash<int, QByteArray> UserModel::roleNames() const
     QHash<int, QByteArray> roles;
     roles[NameRole] = "name";
     roles[ServerRole] = "server";
+    roles[ServerUserStatusRole] = "serverUserStatus";
     roles[StatusIconRole] = "statusIcon";
     roles[StatusMessageRole] = "statusMessage";
+    roles[DesktopNotificationsAllowedRole] = "isDesktopNotificationsAllowed";
```

Comment:
s/isDesktop/desktop/

Yes, they annoyingly went for a different convention for bool between getters and properties...